regression/pages/studio/studio_textbooks.py

```diff
@@ -69,3 +70,14 @@ def delete_all_textbooks(self):
         """
         while self.get_textbook_count() > 0:
             self.click_delete_button()
+
+    def click_view_live_textbook(self):
+        """
+        Clicks View Live button on the first textbook available
+        """
+        button = self.q(css='.view').results[0]
+        # This button is hidden, hovering on it makes it visible
```

Comment:
Don't need to change the test, but I wonder if this is considered an accessibility issue. Did you happen to notice if you could tab to it?